### ISSUE TITLE
Main page flow

### ISSUE THREAD

**Issue by DanVanAtta** [williamdekryger/triplea-game.github.io#5]
First, main page should be geared for a person that just searched for "axis and allies" and is looking for an open source version of the game. So we optimize for their experience when they hit the page.

With that stated, the first page I think should have a bit of a feel similar to: https://pages.github.com/.
Specifically, the page is presented to you in the order in which you need to do things. Namely, try to predict the logical flow a first time user, and present things in that order.

So main page updates I would propose:
- we add the first sentence of our introduction paragraph above the download button. We also will need to clean up that text some as well, make it read better.
- remove the extra DL links, backup mirror and "other downloads". I think three buttons in a row, one for windows installer, mac and then linux installer. Then we'll have another button or link to download, either on the next line or same line for the game manuals

Then scrolling down the page a bit we'll have the map images, then below those we'll kick things off with the second paragraph of the intro. Then we can list the features of the game a bit. Here we can do two things, first we are selling the game on what it offers, secondly, we are already giving someone a tutorial of what they can do to start playing once they have downloaded the game. So a player will *know* how to start up a single player game and play against the AI, and will know where the game manual is and possibly have read it, all before they have even *finished* downloading the game let alone installing it.

**Comment by williamdekryger** [comment]
I agree that that could be a good idea. Any specific ideas on implementation? Right now it looks like any other software download page, imo. Maybe we need to have some non computer savvy A&A.org players do a UI test of some sort, like "Find the Linux download," "Get the answer to 'How to setup port forwarding'," etc.

**Comment by DanVanAtta** [comment]
Simple  static HTML content to start. Eventually javascript could get thrown in maybe.

Good suggestion for the user testing of the site. I'd recommend going further and get non-A&A.org players to test things out (family members for example perhaps). The experiment I would like to see is what someone naturally does when they sit in front of the main page and are not given any prompts or tasks. I would want them to automatically know that they need to find their OS specific installer, download it, install the game, and while they wait for DL I would want them to automatically start learning about the game. Ideally when the game DL is done, a new user will already be pretty savvy and would be able to start a single player game and would almost be able to play maybe..

The installer piece and new user experience though does leave the website, happily there are efforts there. @gaborbernat is working on a better installer, I've been making some attempts at the game engine UX.



My thought behind the main page flow is to try and place items in the path of the users attention no matter what that path turns out to be. I had some opportunity to work with eye tracking studies in the past  (for a very large Seattle based website company), so thinking about some of those results.. Namely a persons attention and focus will generally start somewhere between the top left corner and the center of the screen. A quick scan along that axis is often first then with a quick scan of large block elements (images titles). Focus then often returns back to the center and/or top left where a person then begins to actually *read*, until now only easy to read large header text will have been read (and usually only partially so)

Then, after this initial scan of the page, there are some options of where the users focus and attention will go. If attention focuses back on the center of the page and then starts scrolling vertically, it is a good flow. They get the DL link, DL link to game manual ideally too,  details on then how to play with some helpful overview images, etc. Same thing if the user scans horizontally first, they'll see headers with that same flow "download, learn to play, maps/features, community"

So to summarize what I'm getting at with this last point, we can think of the page as having two independent dimensions, vertical and horizontal. A user could take either one when they are first looking at the page. In either case I'd like us to have a good enough design that we'll get the desired outcome from our users, namely they will download the game and automatically begin their engagement with it.

**Comment by williamdekryger** [comment]
Discussion of content moved to: https://trello.com/c/zRpScrhc
